.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ job_defaults: &job_defaults
     - image: circleci/node:12.9.1
   working_directory: ~/project/repo
 
-cache_key: &cache_key angular-testing-library-deps-cache-{{ checksum "yarn.lock" }}
+cache_key: &cache_key angular-testing-library-deps-cache
```

Comment:
The reason of the lockfile was to cache.
I think that with this change, we should also remove `cache_key` and the cache usage for the CI?
That being said, I was considering to move to GH Actions, so perhaps now would be the perfect time...

Reply:
I don't really know the optimal way to handle this with Circle, so feel free to make changes if you want. Or you can switch to Actions and I'll update this PR to use it instead.